cmd/sid/cli/export.go

```diff
@@ -0,0 +1,125 @@
+package cli
+
+import (
+	"encoding/csv"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/btcsuite/btcd/btcec/v2/schnorr"
+	"github.com/urfave/cli"
+
+	"github.com/babylonlabs-io/staking-indexer/config"
+	"github.com/babylonlabs-io/staking-indexer/indexerstore"
+	"github.com/babylonlabs-io/staking-indexer/utils"
+)
+
+var ExportCommand = cli.Command{
+	Name:  "export",
+	Usage: "Export transactions from the indexer store to a CSV file based on block height.",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  homeFlag,
+			Usage: "Path to the indexer home directory",
+			Value: config.DefaultHomeDir,
+		},
+		cli.StringFlag{
+			Name:  "output",
+			Usage: "Path to the export file",
+			Value: "transactions.csv",
+		},
+		cli.Uint64Flag{
+			Name:  "start-height",
+			Usage: "Start block height for exported transactions",
+			Value: 0,
+		},
+		cli.Uint64Flag{
+			Name:  "end-height",
+			Usage: "End block height for exported transactions",
+			Value: ^uint64(0),
+		},
```

Comment:
I was thinking these could be replaced by 3 arguments as they seem required, i.e., `sid export [start-height] [end-height] [ouput-path]`

Reply:
fixed